integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -133,27 +133,33 @@ public int chmod(String path, @mode_t long mode) {
   /**
    * Changes the owner of an Alluxio file.
    *
-   * This operation only works when the group mapping service is shelled based and the user is
-   * registered in unix. Otherwise it errors as not implemented. This is because the input uid and
-   * gid must match the user and group in Unix.
+   * This operation only works when the shell user group translation is enabled in Alluxio-Fuse,
+   * the user and group are registered in unix, and the user belongs to the group.
+   * Otherwise it errors as not implemented. This is because the input uid and gid
+   * must match the user and group in Unix and should be valid.
    */
   @Override
   public int chown(String path, @uid_t long uid, @gid_t long gid) {
-    if (!mIsShellGroupMapping) {
-      LOG.info("Cannot change the owner of path {} because the group mapping is not shell based",
-          path);
-      // not supported if the group mapping is not shell based
+    if (!mIsUserGroupTranslation) {
+      LOG.info("Cannot change the owner of path {} "
+          + "because the shell user group translation is not enabled in Alluxio-Fuse.", path);
```

Comment:
because the shell -> because shell

Reply:
Changed to 
```
if (!mIsUserGroupTranslation) {
      LOG.info("Cannot change the owner of path {}. Please set {} to be true to enable "
          + "user group translation in Alluxio-Fuse.",
          path, PropertyKey.FUSE_USER_GROUP_TRANSLATION_ENABLED.getName());
      return -ErrorCodes.ENOSYS();
    }
```
to give user more info about what to do!